### ISSUE TITLE
Usage of client.info()

### ISSUE THREAD

**Issue by aylacullen** [AqilCont/alpaca-es6#1]
Can example usage of `client.info()` be provided, please?  I tried using it like this:

```
if(await client.authenticated) {
    // do something i guess
    console.log("i think it works");
    let myinfo = await client.info();
    console.log( JSON.stringify(myinfo) );
  }
```

But it errors out saying `client.info() is not a function`.

**Comment by aylacullen** [comment]
Switched to https://github.com/117/alpaca -- but do feel free to update for others' future reference.

**Comment by aqilc** [comment]
Ah, I just saw this. Didn't show up in my notifs at all!
That property is exactly like `client.authenticated`, it returns a promise even though it's not a method. I probably should have updated the documentation based on that... Will fix.

**Comment by aqilc** [comment]
Sorry, barely anyone used this, so I stopped paying attention to it, but it's still working flawlessly and the JSDoc comments should be enough for anyone using an editor like VSC, so I didn't feel like filling out the `types.d.ts` file. Thanks for opening an issue anyways!

**Comment by aqilc** [comment]
Fixed with [`fee7dca`](https://github.com/AqilCont/alpaca-es6/commit/fee7dca1d14967d7d2d4be29c9ff3b103c0f349a). Closing.
